.github/workflows/ci.yml

```diff
@@ -0,0 +1,58 @@
+name: CI
+
+on:
+    push:
+        branches:
+            - 'master'
+    pull_request:
+    workflow_dispatch:
+
+jobs:
+    validation:
+        name: Validation
+        runs-on: ubuntu-20.04
+
+        steps:
+            - uses: actions/checkout@v2
+
+            - name: Setup PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: '7.4'
+                  extensions: mbstring, intl, bcmath
+                  coverage: none
+
+            - name: Composer Install
+              run: composer install --prefer-dist --no-interaction --profile
+
+            - name: Run validation
+              run: composer validate
+
+            - name: Syntax check
+              run: find ./src -path src -prune -o -type f -name '*.php' -print0 | xargs -0 -n1 -P4 php -l -n | (! grep -v "No syntax errors detected" )
+
+    lowest:
+        name: Prefer Lowest
+        runs-on: ubuntu-20.04
+
+        steps:
+            - uses: actions/checkout@v2
+
+            - name: Setup PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: '7.4'
+                  extensions: mbstring, intl, bcmath
+                  coverage: none
+
+            - name: Composer Install
+              run: composer install --prefer-dist --no-interaction --profile
+
+            - name: Composer Update
+              run: composer update --prefer-lowest --prefer-dist --no-interaction --profile -vvv
+
+            - name: Composer Install prefer lowest tool
+              run: composer require --dev dereuromark/composer-prefer-lowest;
+
+            - name: Run validation
+              run: vendor/bin/validate-prefer-lowest -m
```

Comment:
Shouldn't the workflow be like in other eco modules? They all have PHP 8.1 defined, Composer cache, ...